src/lib/services/api-token-service.test.ts

```diff
@@ -261,4 +261,26 @@ describe('API token getTokenWithCache', () => {
         }
         expect(apiTokenStoreGet).toHaveBeenCalledTimes(2);
     });
+
+    test('should not return the token if it has expired and shoud perform only one db query', async () => {
+        const { apiTokenService, apiTokenStore } = setup({
+            getLogger: () => ({
+                debug: console.log,
```

Comment:
This makes tests very noisy, do we actually want this?

Reply:
Nope! I did that for testing only, removing!